plugins/scenario/World.ts

```diff
@@ -84,7 +84,8 @@ export class World {
   }
 
   async impersonateAddress(address: string, value?: bigint): Promise<SignerWithAddress> {
-    return impersonateAddress(this.deploymentManager, address, value);
+    const dm = this.auxiliaryDeploymentManager ?? this.deploymentManager;
```

Comment:
I think this might break our other scenarios that are impersonating on the main (non-auxiliary) network. WDYT about splitting this into a separate function called `impersonateAddressOnAuxiliary`?

Reply:
agreed; we should be really explicit about which deployment manager we're impersonating from